broker/neb/src/set_log_data.cc

```diff
@@ -243,6 +253,26 @@ bool neb::set_pb_log_data(neb::pb_log_entry& le, const std::string& output) {
   // Duplicate string so that we can split it with strtok_r.
   auto& le_obj = le.mut_obj();
 
+  // fill host_id and service_id on exit
```

Comment:
Could we have more explanations on the interest of this class?
each macro can make a return and this class assures the log to have its host_id, service_id

Reply:
done